app/src/main/java/com/macbitsgoa/ard/utils/NotificationHelper.java

```diff
@@ -0,0 +1,125 @@
+            inboxStyle.addLine(messages[i]);
+
+        int rem = messages.length - 4;
+        if (rem > 0)
+            inboxStyle.setSummaryText("+" + rem + "more..");
+        NotificationCompat.Builder mBuilder = new NotificationCompat.Builder(context)
+                .setStyle(inboxStyle)
+                .setContentTitle(title)
+                .setContentText(context.getString(R.string.newListMessage));
+        mBuilder.setSmallIcon(android.R.mipmap.sym_def_app_icon);
+
+        ///  .addAction("View messages",TODO: pending intent here);
+        notificationManager.notify(random.nextInt(), mBuilder.build());
+    }
+
+    public void bigTextNotif(String title, String message, String summaryText) {
+        NotificationCompat.BigTextStyle bigText = new NotificationCompat.BigTextStyle();
+        bigText.bigText(message);
+        bigText.setBigContentTitle(title);
+        bigText.setSummaryText(summaryText);
+        NotificationCompat.Builder mBuilder =
+                new NotificationCompat.Builder(context)
+                        //     .setSmallIcon(add icon here )
+                        //   .setLargeIcon(add icon here)
+                        .setStyle(bigText)
+                        .setContentText(message.substring(0, 50) + (message.length() > 50 ? "....read more" : ""));
+        mBuilder.setSmallIcon(android.R.mipmap.sym_def_app_icon);
+
+        notificationManager.notify(random.nextInt(), mBuilder.build());
+    }
+
+    public void pictureNotif(Bitmap image, String title) {
+        Uri imageUri = getImageUri(image);
+        Intent intent = new Intent();
+        intent.setType("image/jpeg");
+        intent.setAction(Intent.ACTION_SEND);
+        intent.setData(imageUri);
+        intent.putExtra(Intent.EXTRA_STREAM, imageUri);
+        PendingIntent viewPI = PendingIntent.getActivity(context, random.nextInt(), Intent.createChooser(intent, title), PendingIntent.FLAG_CANCEL_CURRENT);
+
+        intent.setType(Intent.ACTION_VIEW);
+        PendingIntent sharePI = PendingIntent.getActivity(context, random.nextInt(), Intent.createChooser(intent, title), PendingIntent.FLAG_CANCEL_CURRENT);
+
+        NotificationCompat.BigPictureStyle bigPictureStyle = new NotificationCompat.BigPictureStyle();
+        bigPictureStyle.bigPicture(image).build();
+
+        NotificationCompat.Builder mbuilder =
+                new NotificationCompat.Builder(context)
+                        //.setSmallIcon(R.drawable.notification_icon)
+                        .setContentTitle(title)
+                        .setStyle(bigPictureStyle)
+                        .addAction(android.R.drawable.ic_menu_share, "show image", sharePI)  //change icon
+                        .addAction(android.R.drawable.ic_menu_view, "Share", viewPI);                     // change icon
+        mbuilder.setSmallIcon(android.R.mipmap.sym_def_app_icon);
+
+        notificationManager.notify(random.nextInt(), mbuilder.build());
```

Comment:
Fine, disregard my previous request.
We shall see later if this is required to be improved/changed